dirsrvtests/tests/suites/openldap_2_389/migrate_test.py

```diff
@@ -0,0 +1,141 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 William Brown <william@blackhats.net.au>
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import pytest
+import os
+from lib389.topologies import topology_st
+from lib389.password_plugins import PBKDF2Plugin
+from lib389.utils import ds_is_older
+from lib389.migrate.openldap.config import olConfig
+from lib389.migrate.openldap.config import olOverlayType
+from lib389.migrate.plan import Migration
+# from lib389.migrate.plan import *
+
+pytestmark = pytest.mark.tier1
+
+DATADIR1 = os.path.join(os.path.dirname(__file__), '../../data/openldap_2_389/1/')
+
+@pytest.mark.skipif(ds_is_older('1.4.3'), reason="Not implemented")
+def test_parse_openldap_slapdd():
+    """Test parsing an example openldap configuration. We should be able to
+    at least determine the backends, what overlays they have, and some other
+    minimal amount.
+
+    :id: b0061ab0-fff4-45c6-b6c6-171ca3d2dfbc
+    :setup: Data directory with an openldap config directory.
+    :steps:
+        1. 
```

Comment:
Could you please put some steps here? I think some automatic test-doc systems may complain if the fields are empty.